microcosm/tracing.py

```diff
@@ -0,0 +1,35 @@
+from contextlib import nullcontext
+
+from jaeger_client import Config
+
+from microcosm.api import binding, defaults, typed
+
+
+@binding("tracer")
+@defaults(
+    sample_type="ratelimiting",
+    sample_param=typed(int, 10),
+)
+def configure_tracing(graph):
+    """
+    See https://www.jaegertracing.io/docs/1.12/sampling/ for more info about
+    available sampling strategies.
+
+    """
+    if graph.metadata.testing:
```

Comment:
This keeps unit tests from acting up.  Running things locally in `debug` mode can still produce traces. It's UDP so nothing should go wrong if there's no where to send.